test/controllers/global_stats_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+class Api::GlobalStatsControllerTest < ActionDispatch::IntegrationTest
+  # Test Successful Responses
```

Comment:
nitpick but we don't need this comment.

Reply:
Good catch, made the comments while pseudocoding the tests. Removed :+1: